Makefile

```diff
@@ -303,11 +303,21 @@ lint-other-targets:
 	--all-features \
 	-- -D warnings
 
+lint-codespell: ensure-codespell
+	codespell
+
+ensure-codespell:
+	@if ! command -v codespell &> /dev/null; then \
+        echo "codespell not found. Installing..."; \
+        pip install codespell; \
```

Comment:
don't autoinstall it, better to not assume how people want to install it.

would just tell people how they can install it instead and fail if it's not available

Reply:
modified, ptal